Tensile/KernelWriter.py

```diff
@@ -498,6 +499,8 @@ def makeSubIterSchedule(self, kernel, localReadCode, iteration, pointerCode, wai
               # if there is localWrite at first mfma, need to skip it in waitcnt.
               if i == 0:
                 skipLocalWriteWaitcnt += writeItem.countType(Code.LocalWriteInst)
+              if not localReadItems:
+                self.perIterLocalWriteCanSkip[iteration] += writeItem.countType(Code.LocalWriteInst)
```

Comment:
yes

skipLocalWriteWaitcnt: it counts localWrites in current iteration, since some of localWrites are interleaved with mfma, we only count localWrites before waitcnt ( i==0 )

perIterLocalWriteCanSkip[]: it counts localWrites after last localReads is scheduled in each iteration.